src/connectedmachine/azext_connectedmachine/aaz/latest/connectedmachine/license/_create.py

```diff
@@ -0,0 +1,402 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "connectedmachine license create",
+    is_preview=True,
+)
+class Create(AAZCommand):
+    """Create operation to create a license.
+
+    :example: Sample command for license create
+        az connectedmachine license create --name 'myESULicense' --resource-group 'ytongtest' --location 'eastus2euap' --license-type 'ESU' --state 'Activated' --target 'Windows Server 2012' --edition 'Datacenter' --type 'pCore' --processors 16 --subscription 'b24cc8ee-df4f-48ac-94cf-46edf36b0fae'
```

Comment:
```suggestion
--subscription 'b24cc8ee-df4f-48ac-94cf-46edf36b0fae'
```
Please use the mock subscription instead of a real subscription

Reply:
resolved